src/test/java/org/beanpod/switchboard/controller/EncoderControllerTest.java

```diff
@@ -142,12 +148,28 @@ final void testUpdateEncoder() {
   @Test
   final void testUpdateEncoderExceptions() {
     EncoderDTO encoderDto = EncoderFixture.getEncoderDto();
-    when(encoderService.findEncoder(encoderDto.getSerialNumber())).thenReturn(Optional.empty());
+    when(encoderDao.findEncoder(encoderDto.getSerialNumber())).thenReturn(Optional.empty());
 
     assertThrows(
         ExceptionType.DeviceNotFoundException.class,
         () -> {
           encoderController.updateEncoder(encoderDto);
         });
   }
+
+  @Test
+  final void testGetEncoderStreams() {
+    when(encoderService.getEncoderStreams(any(String.class)))
+        .thenReturn(List.of(StreamFixture.getStreamDto()));
+    when(streamMapper.toModel(anyList())).thenReturn(List.of(StreamFixture.getStreamModel()));
```

Comment:
I like to use the explicit call values when mocking instead of `any` because it helps catch more errors when changes are made

Reply:
The reason I did this instead of putting explicit call values is that if the value changes, the method will return null instead and the error message becomes less evident.

Like this, if you call with the wrong thing, then it will fail at the stage of the `verify` with a nice little diff telling you what properties don't match.

Reply:
That's fair